boards/arm/control_freak/control_freak.dts

```diff
@@ -0,0 +1,409 @@
+/dts-v1/;
+#include <st/f4/stm32f446Xe.dtsi>
+#include <st/f4/stm32f446r(c-e)tx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+    model = "Control Freak";
+    compatible = "launch,control_freak";
+
+    chosen {
+        zephyr,sram = &sram0;
+        zephyr,flash = &flash0;
+
+        zephyr,console = &usart1;
+        zephyr,shell-uart = &usart1;
+
+    };
+    aliases {
+        debug-serial = &uart4;
+        led0 = &led1; // a little broken :(
+        buzz = &buzzer;
+        
+        gnss = &gnss;
+        gpsreset = &gps_reset;
+        gpssafeboot = &gps_timepulse;
+        lora0 = &lora;
+        servo1 = &servo1;
+        servo2 = &servo2;
+        servo3 = &servo3;
+        radioreset = &radioreset;
+    };
+
+    servos: servos{
+        status="okay";
+        compatible = "pwm-leds";
+        min-pulse = <PWM_USEC(100)>;
+        max-pulse = <PWM_USEC(3000)>;
+
+        servo1: servo1 {
+            label = "Servo 1 PWM";
+            // &gpioc 6 GPIO_ACTIVE_HIGH
+            pwms = <&pwm3 1 PWM_MSEC(3) PWM_POLARITY_NORMAL>;
+        };
+        servo2: servo2 {
+            label = "Servo 2 PWM";
+            // &gpioc 7 GPIO_ACTIVE_HIGH
+            pwms = <&pwm3 2 PWM_MSEC(3) PWM_POLARITY_NORMAL>;
+        };
+        servo3: servo3 {
+            label = "Servo 3 PWM";
+            // &gpiob 15 GPIO_ACTIVE_HIGH
+            pwms = <&pwm12 2 PWM_MSEC(3) PWM_POLARITY_NORMAL>;
+        };
+    };
+
+    leds: leds {
+        compatible = "gpio-leds";
+        led1: led1{
+            // DOESNT WORK, TRANSISTOR ON BACKWARDS
+            // GOES FROM ON TO MORE ON
+            gpios = <&gpioc 11 GPIO_ACTIVE_HIGH>;
+            label = "GPS LED";
+        };
+
+        radioreset: radioreset{
+            gpios = <&gpioa 4 GPIO_ACTIVE_LOW>;
+            label ="Radio Reset";
+        };
```

Comment:
Shouldn't this be handled by the driver itself?

Reply:
Yes, but i wanted an extra handle for it when going behind the drivers back to do horus